aws/rds_types.go

```diff
@@ -0,0 +1,8 @@
+package aws
+
+type RdsDeleteError struct{}
```

Comment:
This error struct should probably have a parameter indicating which RDS instance failed (preferably ID). That way, users can investigate later what RDS instance failed to delete, whether it is systematic, or if `cloud-nuke` just didn't wait long enough.

Reply:
This type is only being used on a function at `rds_test`. Anyway I improved the error message used on the main function to return the instance's name.

Reply:
Ah gotcha! It still helps to include the instance ID though so that we can debug the test failure. With something like cloud-nuke tests where we constantly spin up and down RDS instances, it can be difficult to know which one was associated with which test failure.